setup.py

```diff
@@ -141,8 +141,10 @@ def read_version_py(infname):
 
     # HTTP server and client
     "klein",
+
     # 2.2.0 has a bug: https://github.com/pallets/werkzeug/issues/2465
-    "werkzeug != 2.2.0",
+    # 2.3.x has an incompatibility with Klein: https://github.com/twisted/klein/pull/575
+    "werkzeug != 2.2.0, != 2.3.0, != 2.3.1",
```

Comment:
This works around the incompatibility that motivated the work in the first place.  Since the Docker image wheelhouses are populated from the declared dependencies, this lets us fix CI by making it possible to generate new Docker images (which I did last week) that don't include the incompatible version of werkzeug.

Reply:
I don't think this is sufficient:

* 2.3.2 just came out, presumably there will be 2.3.3.
* The issue isn't a bug in werkzeug, the issue is that they removed deprecated support for passing in bytes, no version post 2.2 will work.

I suggest `< 2.3` instead.